### ISSUE TITLE
Eslint/Tslint error on @types

### ISSUE THREAD

**Issue by stephenjason89** [vue-styleguidist/vue-styleguidist#1195]
Documenting a property with boolean causes eslint warning because prop is technically an object with type Boolean
![image](https://user-images.githubusercontent.com/3810177/132830943-451847d7-69aa-4d00-8e3f-2b21ae98f277.png)

Here's the error
![image](https://user-images.githubusercontent.com/3810177/132831095-58541631-5b1d-4f0d-9cae-443bccd2109e.png)

**Comment by elevatebart** [comment]
Hello @stephenjason89,

You are right, it could be a problem. Fortunately for you, you should not need to add the `@type` flag, styleguidist should detect it automagically.

Try this instead, it should render the style guide you want:

```js
  /**
   * Loading status of modal component
   */
  loading: {
    type: Boolean
    default: false
  }
```

**Comment by stephenjason89** [comment]
Thank you @elevatebart. Should I keep this open or close this as this can still be a slight problem on other scenarios maybe to someone who want to be more verbose. 

I will be removing all my @types on my props ☺️ I like it cleaner. 

Thank you for the tip
